pkg/azure/aks.go

```diff
@@ -101,6 +101,28 @@ func NewCluster(ctx context.Context, subscriptionId, resourceGroup, name, locati
 	return nil
 }
 
```

Comment:
need to add a get managed cluster method so we can get a single cluster

Reply:
i added this on my PR, so we don't need it here

Reply:
indeed